app/src/main/java/com/dmonzonis/nookpendium/RecordListAdapter.kt

```diff
@@ -33,7 +33,7 @@ class RecordListAdapter(private val records: List<Record>) :
     override fun onBindViewHolder(holder: RecordHolder, position: Int) {
         val record: Record = records[position]
         holder.itemView.text_name.text = record.name
-        holder.itemView.text_price.text = record.price
+        holder.itemView.text_price.text = record.price?.toString() ?: "?"
```

Comment:
Does this still happen with our data set or is this a safeguard for just in case?

Reply:
Doesn't happen with the latest data but I left it just in case